### PR TITLE
Dynamic fees transaction with javascript

### PR DESCRIPTION
**Pull request by rajranjan0608**

### CODE REVIEW

File: docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+| **undefined** | 2                | It will calculate the max fee by adding the provided priority fee with the estimated base fee. Take extra precaution here as the max fee will now be capped by `baseFee + priorityFee`, which can consume all the provided priority fees. |
+| 100       | **undefined**        | It will estimate the priority fee and use the provided max fee. If the estimated priority fee is more than the provided max fee, then it throws an error.                                                                            |
+| **undefined** | **undefined**        | It will estimate the base fee and priority fee from the network, and will add both the values to calculate the max fee per gas. Again, you have to pay whatever will be estimated.                                               |
+
+You will get the following output on the successful submission of the signed transactions. Using this URL you can view the status of your transaction on Snowtrace.
+```bash
+View transaction with nonce 25: https://testnet.snowtrace.io/tx/0xd5b92b85beaf283fbaeeefb95c9a17a6b346a05b6f9687f2d6e421aa79243b35
+```
+
+## Reissuance of stucked transaction
```
Comment (yulin-dong) [inline docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md:179]:
```suggestion
## Reissuance of Stuck Transaction
```

File: docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+Sometimes during high network activity, all transactions couldn't make it to the latest blocks for a long time, due to relatively lower effective tip than the other transactions in the pool. We can either re-issue the same transaction with a higher priority fee or cancel the transaction. To re-issue the stucked transaction, you can send a new one with same amount and data but higher priority fee and same nonce value as the stucked transaction. The transaction with lower effective tip will automatically be rejected (due to same nonce), and you do not need to worry about it. You can also cancel the stucked transaction, by keeping the amount to 0, with a higher priority fee and same nonce. Let's say, the above transaction with a nonce value of 25 has stucked. You can then re-issue a new transaction with same nonce, but higher priority fee this time.
+
```
Comment (yulin-dong) [inline docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md:182]: I think we should use stuck, not stucked, https://jakubmarian.com/the-word-stucked-in-english/

**Review by yulin-dong — CHANGES_REQUESTED** [review]

File: docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md (hunk 2 above)
Comment (rajranjan0608) [reply docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md:182]: lol. Will fix this.

File: docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+		value: ethers.utils.parseEther(amount),
+		chainId,
+	};
+
+	tx.gasLimit = await HTTPSProvider.estimateGas(tx);
+
+	const signedTx = await wallet.signTransaction(tx); 
+	const txHash = ethers.utils.keccak256(signedTx);
+
+	console.log(`View transaction with nonce ${nonce}: https://testnet.snowtrace.io/tx/${txHash}\n`);
```
[the remark this answers is not in the document]
Comment (rajranjan0608) [reply docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md]: Yes.

File: docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+	// Sending a signed transaction and waiting for its inclusion
+	await (await HTTPSProvider.sendTransaction(signedTx)).wait();
+};
+```
+
+This function calculates transaction hash from the signed transaction and logs on the console, the URL for transaction status on the Snowtrace explorer.
```
[the remark this answers is not in the document]
Comment (rajranjan0608) [reply docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md:152]: I thought adding this line after the code would explain it better.

**Review by yulin-dong — APPROVED** [review]

File: docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (cgcardona) [reply docs/build/tutorials/platform/sending-transactions-with-dynamic-fees-using-javascript.md:152]: Ok

**Pull request merged**